src/distribution/binomial.rs

```diff
@@ -293,18 +289,15 @@ impl Discrete<u64, f64> for Binomial {
     /// Calculates the probability mass function for the binomial
     /// distribution at `x`
     ///
-    /// # Panics
-    ///
-    /// If `x > n`
-    ///
     /// # Formula
     ///
     /// ```ignore
     /// (n choose k) * p^k * (1 - p)^(n - k)
     /// ```
     fn pmf(&self, x: u64) -> f64 {
-        assert!(x <= self.n,
-                format!("{}", StatsError::ArgLte("x", 1.0)));
+		if x > self.n {
```

Comment:
No, the opposite should happen: the match should go away. Matching on floating-point values [is deprecated](https://github.com/rust-lang/rust/issues/41255). It's currently only a warning but it will become a hard error in the future.